app/build.gradle

```diff
@@ -23,10 +23,16 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    dataBinding{
+        enabled=true
```

Comment:
We care about our code and his readability so each space it's important. 

Find more about shortcuts in android studio and which is the one to help you format your code.

Reply:
I used CTRL+ALT+L